sdk/servicebus/Azure.Messaging.ServiceBus/src/Processor/ServiceBusProcessorOptions.cs

```diff
@@ -79,7 +77,7 @@ public TimeSpan MaxAutoLockRenewalDuration
             {
                 if (value.HasValue)
                 {
-                    TimeoutHelper.ThrowIfNegativeArgument(value.Value, nameof(MaxReceiveWaitTime));
+                    Argument.AssertPositive(value.Value, nameof(MaxReceiveWaitTime));
```

Comment:
This should actually be positive rather than just non-negative.

Reply:
I assume the result of `Argument.AssertNotNegative` and `Argument.AssertPositive` will be the same, both will throw for negative value? if yes, then why do have 2 different methods?

Reply:
AssertNotNegative allows >= 0
AssertPositive allows > 0